src/translations/maps/queuedPhysicalIndexToValueMap.js

```diff
@@ -0,0 +1,103 @@
+import IndexMap from './indexMap';
+import { getListWithRemovedItems, getListWithInsertedItems } from './utils/physicallyIndexed';
+import { getDecreasedIndexes, getIncreasedIndexes } from './utils/actionsOnIndexes';
+import { isFunction } from '../../helpers/function';
+
+/**
+ * Map for storing mappings from an physical index to a value. Some values are stored in a certain order.
+ *
+ * Does not update stored values on remove/add row or column action. Otherwise, queue of indexes related to ordered
+ * values is updated after such changes.
+ */
+class QueuedPhysicalIndexToValueMap extends IndexMap {
+  constructor() {
+    super();
+    this.queueOfIndexes = [];
+  }
```

Comment:
```suggestion
queueOfIndexes = [];
```

It could be simplified by moving the prop declaration to the class fields. Then no constructor recreation is necessary. And I'd add documentation to that field.

Reply:
Done within: https://github.com/handsontable/handsontable/commit/3ac658d899570aa90829e5f048d9ce038808ed83.